src/table/column.rs

```diff
@@ -1,3 +1,5 @@
+use std::fmt::Display;
```

Comment:
I prefer to write like: `use std::fmt` and then write everywhere with prefix

Reply:
made the requested changes